rs/bitcoin/kyt/src/state.rs

```diff
@@ -52,26 +52,26 @@ pub struct FetchedTx {
 // Max number of concurrent http outcalls.
 const MAX_CONCURRENT: u32 = 50;
 
-// Max number of entries in the cache is set to 10_000. Since the average transaction size
-// is about 400 bytes, the estimated memory usage of the cache is in the order of 10s of MBs.
-const MAX_FETCH_TX_ENTRIES: usize = 10_000;
+// Max number of entries in the cache is set to 3000 so that dashboard html has a good chance to
```

Comment:
It would be nice to describe a bit more precisely what "good chance" means.

Reply:
I think pagination is a better solution to this problem (of avoiding 2mb limit).

Reply:
Reverted this back to 10_000, and implemented pagination with a test.